device/common/include/traccc/seeding/device/weight_updating.hpp

```diff
@@ -0,0 +1,34 @@
+// Project include(s).
```

Comment:
Add the common copyright statement to all new files.

Also, add

```c++
#pragma once
```

to all of the header files. (Checking again, you only seem to have forgotten about both of them for this one file. :stuck_out_tongue:)